public/views/block.html

```diff
@@ -59,7 +59,8 @@ <h2 translate>Summary</h2>
           </tr>
           <tr>
             <td><strong translate>Block Reward</strong></td>
-            <td class="text-right text-muted">{{$root.currency.getConvertion(block.reward) || block.reward + ' DCR'}}</td>
+            <td data-ng-show="block.height != 0" class="text-right text-muted">{{block.reward + ' DCR'}}</td>
```

Comment:
The height check is correct but this probably still needs:
 {{$root.currency.getConvertion(block.reward) || block.reward + ' DCR'}}